guide/packages.md

```diff
@@ -134,8 +136,8 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
 | `membrane_mpegaudio_format` | MPEG audio format definition                   | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_mpegaudio_format.svg)](https://hex.pm/packages/membrane_mpegaudio_format) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_mpegaudio_format) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_mpegaudio_format) |
 | `membrane_caps_audio_raw`   | Raw audio format definition                    | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_caps_audio_raw.svg)](https://hex.pm/packages/membrane_caps_audio_raw) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_caps_audio_raw) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-caps-audio-raw)         |
-| `membrane_h264_format`  | H264 video format definition                   | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_h264_format.svg)](https://hex.pm/packages/membrane_h264_format) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_h264_format/) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_h264_format)     |
-| `membrane_raw_video_format`   | Raw video format definition                    | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_raw_video_format.svg)](https://hex.pm/packages/membrane_raw_video_format) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_raw_video_format) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_raw_video_format)         |
+| `membrane_h264_format`      | H264 video format definition                   | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_h264_format.svg)](https://hex.pm/packages/membrane_h264_format) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_h264_format) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-h264-format)                     |
+| `membrane_caps_video_raw`   | Raw video format definition                    | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_caps_video_raw.svg)](https://hex.pm/packages/membrane_caps_video_raw) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_caps_video_raw) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane-caps-video-raw)         |
```

Comment:
Why those changes? I think previous version was the correct one.

Reply:
Yup the previous version was the correct one, I probably made a mistake during rebase 😅 .